### PR TITLE
Retroactively Populate Unique IDs

### PR DESCRIPTION
**Pull request by sbassah**
Original Gist: https://gist.github.com/spivurno/535b74a34c0830adf1b770fed0de7812

### CODE REVIEW

File: gp-unique-id/gpuid-retroactively-populate-unique-ids.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ *
+ *     Make sure you replace "123" with the ID of your form. Full URL should look like this:
+ *
+ *     http://mysite.com/?gpui_retro_pop=1&form_id=123
+ *
+ * 3. Submit the updated URL. If everything is correct, you will be greeted with a success message and updated entry count.
+ *
+ * NOTE: This snippet has a hard-coded limit of 999 entries.
+ *
+ * Plugin Name: GP Unique ID  - Retroactively Populate Unique IDs
```
Comment (spivurno) [inline gp-unique-id/gpuid-retroactively-populate-unique-ids.php:22]: Remove extra whitespace from plugin name:
> GP Unique ID - Retroactively Populate Unique IDs

File: gp-unique-id/gpuid-retroactively-populate-unique-ids.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Gravity Perks // GP Unique ID // Retroactively Populate Unique IDs
+ * https://gravitywiz.com/documentation/gp-unique-id/
+ *
+ * Retroactively populate unique IDs into Unique ID fields. Useful when you want to regenerate the unique IDs within the form entries.
+ * Usage:
```
Comment (spivurno) [inline gp-unique-id/gpuid-retroactively-populate-unique-ids.php:7]: Add an additional line break to separate the long description from the instructions.

**Review by spivurno — APPROVED** [review]
Approved with minor tweaks.

**Comment by sbassah** [comment]
@spivurno It appears this snippet has already been added to the Library. So I am closing this PR.
https://github.com/gravitywiz/snippet-library/blob/master/gp-unique-id/gpuid-retroactively-populate-unique-ids.php

**Pull request closed without merging**